src/shared/selectors/flags.ts

```diff
@@ -70,6 +71,7 @@ export const CLOUD_FLAGS = {
   'flow-panel--raw-flux': true,
   'flow-panel--to-bucket': true,
   'flow-panel--notification': true,
+  'flow-panel--remote-csv': true,
```

Comment:
should these be defaults as false until we get to test them out? By setting this to true without a feature flag to set in config cat, it'll be on by default and any issues that arise will have to be addressed by sending a PR in the UI to set this to false

Reply:
sounds like a plan